terraform/README.md

```diff
@@ -76,6 +95,14 @@ Follow the set up guide outlined here: https://github.gds/gds/digitalmarketplace
 
 The Makefile will look for the credentials repository at the directory defined in the DM_CREDENTIALS_REPO environment variable.
 
+You may like to add this to your `.envrc` file for easy access.
```

Comment:
Or `.bash_profile`/equivalent so that you don't need to define it in multiple places.